spec/controllers/redirect_controller_spec.rb

```diff
@@ -20,7 +16,7 @@
   describe "#item" do
     it "redirects to item page" do
       get :item, uuid: fedora3uuid1
-      expect(response).to redirect_to "http://test.host/files/#{gf.id}"
+      expect(response).to redirect_to generic_file_path(gf.id)
```

Comment:
One small thought about this (and please excuse any newbie ignorance):  why not take it one step further and use generic_file_path(gf) (excluding the id method). Then the test becomes immune to "to_param" being overridden in the model.